manifests/o/OpenJS/NodeJS/12.17.0/OpenJS.NodeJS.yaml

```
@@ -1,8 +1,8 @@
-PackageIdentifier: OpenJS.Nodejs
+PackageIdentifier: OpenJS.NodeJS
 PackageVersion: 12.17.0
 PackageName: Node.js
 Publisher: OpenJS Foundation
```

Ugh more Publisher issues
```suggestion
Publisher: Open.js Foundation
```